### DISCUSSION TITLE
Plan device to replace decommissioning device

### DISCUSSION THREAD

**Discussion by RichardCrooks** [netbox-community/netbox#14902] [Help Wanted!] [answered]
Hello, 

I'm pretty new to Netbox, and can't find the option I need (if it exists).

I have a device in Rack A2 VU39 which is set to decommissioning. I'd like to plan in a new device to replace it however, I cannot select VU39 as it's allocated to the device due for decommissioning.

Is there an option that allows for planned devices to occupy the same VU as a device being decommissioned?

Cheers,
Rich

**Comment by atownson** [comment, marked as answer]
Hey Rich,
I might suggest creating the planned device without entering the rack values (as a non-racked device). Then once the replacement occurs, delete or remove the rack values for the legacy device and update the rack values for the replacement device. There's not a planned-change mechanism for racking devices other than reserving unused spaces.

**Reply by RichardCrooks** [reply]
After some more digging I think your solution is exactly what I need. Thanks for the suggestion!

**Comment by xkilian** [comment]
You would also need to note where this planned unracked device goes.
Maybe a custom field for the planned U position.
Like for planned connections I would rather be able to attach ot to the destination interface or U. It is much clearer and dos not involve a kludgey custom field or description or home brew documentation.
There is an issue for exactly this but on interfaces, same could apply for U space.
Cheers
